lib/text/ttml_text_parser.js

```diff
@@ -110,14 +112,16 @@ shaka.text.TtmlTextParser.prototype.parseMedia = function(data, time) {
     let rateInfo = new TtmlTextParser.RateInfo_(
         frameRate, subFrameRate, frameRateMultiplier, tickRate);
 
+    let metadataElements = TtmlTextParser.getLeafNodes_(
```

Comment:
General note, our coding guidelines now require all variables that can be `const` must be `const`. You don't need to update the surrounding variables, but any variable you add/change, need to be updated to follow this rule.